libgraphics/source/VulkanApplication/VulkanApplication.cxx

```diff
@@ -125,7 +125,7 @@ try {
     vk_utils::vk_try(device.waitForFences(frame.inFlightFences, VK_TRUE, UINT64_MAX));
 
     uint32_t imageIndex = swapchain.getNextImageIndex(UINT64_MAX, frame.imageAvailableSemaphore);
-    allocator.setCurrentFrameIndex(imageIndex);
+    // allocator.setCurrentFrameIndex(imageIndex);
```

Comment:
Either remove the line, or add a comment explaining when this line should be re-enabled.